### DISCUSSION TITLE
Changing size of the command line.

### DISCUSSION THREAD

**Discussion by aayan-baig** [CosmosOS/Cosmos#2119] [Q&A]
<h3>Hi! I'd like to make the size of my command line bigger in VMware. It's pretty small.</h3>

Anyone know how to? Lmk if you do, thanks a lot! 

<h1>ORIGINAL SIZE:</h1>
![image](https://user-images.githubusercontent.com/95202769/153760563-fe31b572-b0de-4a48-9c35-74841a116042.png)

**Comment by ghost** [comment]
Console.SetWindowSize(Int32, Int32) should do the trick. Keep in mind only a few resolutions are supported.
You can see all the width & height combinations here: https://github.com/CosmosOS/Cosmos/blob/master/source/Cosmos.System2_Plugs/System/ConsoleImpl.cs#L571
